src/css/library-vars.css

```diff
@@ -229,6 +230,13 @@ leaving room for semantic and accessible implementation choices  */
     --icon-sprite: var(--fa-search);
 }
 
+.icon.fa-close, .icon-attach.fa-close:before {
+    --icon-sprite: var(--fa-close);
+}
+
+.icon-replace.fa-instagram {
```

Comment:
This appears to be erroneous?

Reply:
@possumbilities my thought was to accomodate for both an `.icon-attach` and an `.icon-replace` instance. But if the `.icon-attach` usecase is not necessary, I can remove it

Reply:
I meant it looks like its a duplicate of `fa-instagram` even those this PR is for `fa-close`?

Reply:
it's fine to have attach and replace for icon-close. It was just the `instagram` duplication mistake that needs addressing :)

Reply:
Mr Quick fingers has removed the attach implementation 🥲.

I have removed the duplicate `instrgram` implementation

Reply:
all good now :)